src/IROperator.h

```diff
@@ -820,7 +885,12 @@ inline Expr pow(Expr x, Expr y) {
     if (x.type() == Float(64)) {
         y = cast<double>(y);
         return Internal::Call::make(Float(64), "pow_f64", {x, y}, Internal::Call::Extern);
-    } else {
+    }
+    else if (x.type() == Float(16)) {
+         y = cast<float16_t>(y);
+        return Internal::Call::make(Float(16), "pow_f16", {x, y}, Internal::Call::Extern);
```

Comment:
How would you tell the backend to do truncation? I recently discovered that the LLVM ``fptrunc`` instruction doesn't actually truncate!

http://lists.llvm.org/pipermail/llvm-dev/2015-August/089504.html